static/admin/config.yml

```diff
@@ -30,14 +30,14 @@ collections:
           - { label: "Path", name: "path", widget: "hidden", default: "/" }
           - {
               label: "SEO Title",
-              name: "title",
+              name: "seoTitle",
               widget: "string",
               hint: "SEO title",
               required: false,
             }
           - {
               label: "SEO Description",
-              name: "description",
+              name: "seoDescription",
```

Comment:
We renamed the `title` to `seoTitle` and `description ` to `seoDescription` for every page so we should update the `md` files(let's replace the `title` with `seoTitle` and `description` with `seoDescription` or add a new seo fields if the `title` and `description` are needed in page components) and in the `gatsby-node.ts` file here https://github.com/threshold-network/splash-page/blob/threshold-site-2.0/gatsby-node.ts#L56-L57.